storage/fio-test-setup.yaml

```diff
@@ -96,21 +100,29 @@
       --test-types=read,write,rw
       --clients={{ fio_servers }}
       --config=SEQ_IO
-      --samples=3
+      --samples=1
       --max-stddev=20
-      --block-sizes=4,128,4096
+      --block-sizes=4,16,64
       --job-file=/tmp/sequential_io.job
+      --pre-iteration-script=/tmp/drop-cache.sh
```

Comment:
We could use the script directly which is already in our svt repo on master.
In that way, we could remove the copy task above.